web/src/main/java/org/akaza/openclinica/controller/helper/RestfulServiceHelper.java

```diff
@@ -0,0 +1,80 @@
+package org.akaza.openclinica.controller.helper;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class RestfulServiceHelper {
+	
+	//CSV file header	
+	private static final String [] FILE_HEADER_MAPPING = {"ParticipantID"};
+	
+	     
+
+	
+	/**
+	 * @param file
+	 * @return
+	 * @throws IOException
+	 */
+	public static ArrayList<String> readCSVFile(MultipartFile file) throws IOException {
+		
+		 BufferedReader reader;
+		 ArrayList<String> subjectKeyList = new ArrayList<>();
+		 String line;
+		 InputStream is = file.getInputStream();
```

Comment:
use the try block so that stream closes properly as here:
https://baptiste-wicht.com/posts/2010/08/java-7-try-with-resources-statement.html